### ISSUE TITLE
DH mod for Portuguese (Brazil) Layout

### ISSUE THREAD

**Issue by Khrona98** [DreymaR/BigBagKbdTrixXKB#27]
Hello there!

I noticed that the US variant of the Colemak layout have a DH version, but currently there is no DH variant for Portuguese (Brazil) for both `keep local symbols` and `unified symbols`.

This can be circumvented by ticking the `Colemak Curl (DH) ergo Mod` option inside `Miscellaneous compatibility options` in the `Advanced tab` on the keyboard setting on KDE Plasma (Screenshot below), but I game a lot and Wine/Proton does not support dynamically changing your keyboard layout, so it defaults to the first one in the system settings. Having a DH layout in Portugese for both variants I listed above would fix this since there would be no need to keep Colemak as the first layout on the list.

<details>
  <summary>Screenshot</summary>

![Screenshot_20220326_144125](https://user-images.githubusercontent.com/9277125/160251315-f464a5e6-1475-44c8-a3a1-8e092aff0312.png)

</details>

**Comment by DreymaR** [comment]
Ah, yes. I've let the Curl(DH) mod be an option so far, but that's not enough anymore. I'm very busy with some other stuff atm, but I promise I'll try to find the time to add Colemak-DH as proper layouts everywhere. I hope users manage to set the keyboard model component right for their case then... You know that with a row-staggered board you should use an Angle mod for proper Colemak-DH?

**Comment by DreymaR** [comment]
Is that KDE you're using? I think that GNOME went stupid at some point and removed the keyboard model setting from the UI.

**Comment by Khrona98** [comment]
Yep it's KDE. I find the keyboard settings in there to be pretty good.

Unfortunately Gnome does have a history of stripping features away. Although there's probably ways around that with some GUI.
